### DISCUSSION TITLE
Add maintenance window to supress alerts

### DISCUSSION THREAD

**Discussion by MorganMLGman** [henrygd/beszel#265] [Ideas]
Hi,

I think it would be beneficial to add something like maintenance window or maintenance switch that will temporarily disable sending alerts when triggers are activated. 

Maintenance window with start_date_time and end_date_time would be better than switch as switch could be forgotten and left on.

This should be possible to set per machine or global. Stats should still be collected, but any alarm triggers will be suppressed during maintenance.

**Comment by silmarine** [comment]
I came here to request this feature. Right now I have to manually pause each machine monitor. Luckily all I have is 5 machines but it's still annoying. It would also be nice if you didn't have to set a time but just an activate maintenance mode button or pause all machines that then needs to be manually turned off.

**Comment by henrygd** [comment]
I'll try to get an option added for this in the next feature release or two.

**Comment by Nexulo** [comment]
What is the current status of this feature's development?

**Comment by Cheezzhead** [comment]
This is the only feature that's really still missing for me to use Beszel as my primary monitoring tool. At present, my nightly maintenance window (local backups, cloud backups, cleanups, scripts, etc...) triggers several alerts for things like load or cpu usage, even though they're expected to be high at that moment. The only way to resolve this now is to increase the trigger values, but this is far from ideal (actually kind of the antithesis of server monitoring imo) because outside of maintenance tasks they're very sensible limits.

**Comment by henrygd** [comment]
Forgot about this, will move it near the top of my list.

**Comment by henrygd** [comment]
This is now in 0.17.0 as "Quiet Hours," located at `/settings/notifications`.

Let me know if it doesn't work properly for you :+1:
